tests/test_audit_search.py

```diff
@@ -0,0 +1,94 @@
+from tests import factories
+from tests.common import ApiBaseTest
+
+from webservices.resources.auditsearch import AuditSearchView
+from webservices.rest import api
+
+
+class TestAuditSearch(ApiBaseTest):
+
+    def test_audit_id_fetch(self):
+        """ check if the specified audit_id exist """
+        audit_id = 565
+        factories.AuditSearchViewFactory(audit_id=audit_id)
+
+        results = self._results(api.url_for(AuditSearchView, audit_id=audit_id))
+        self.assertEqual(results[0]['audit_id'], audit_id)
+
+    def test_rad(self):
+        """ Check Audit Search returns general endpoint"""
+        factories.AuditSearchViewFactory(committee_id='C00241083')
+        factories.AuditSearchViewFactory(committee_id='C00135558')
+
+        results = self._results(api.url_for(AuditSearchView))
+        self.assertEqual(len(results), 2)
+
+    def test_filters(self):
+        [
+            factories.AuditSearchViewFactory(finding_id=3),
+            factories.AuditSearchViewFactory(finding='Disclosure'),
+            factories.AuditSearchViewFactory(issue_id=220),
+            factories.AuditSearchViewFactory(issue='Coordinated Expenditures'),
+            factories.AuditSearchViewFactory(election_cycle=2002),
+            factories.AuditSearchViewFactory(committee_id='C00161786'),
+            factories.AuditSearchViewFactory(committee_name='COLORADO DEMOCRATIC PARTY'),
+            factories.AuditSearchViewFactory(committee_designation='U'),
+            factories.AuditSearchViewFactory(committee_type='Y'),
+            factories.AuditSearchViewFactory(committee_description='Party'),
+            factories.AuditSearchViewFactory(candidate_id=''),
+            factories.AuditSearchViewFactory(candidate_name=''),
+            # we may have to add filters and indexes to these fields???
+            # factories.AuditSearchViewFactory(audit_case_id=1203),
+            # factories.AuditSearchViewFactory(link_to_report='http://transition.fec.gov/audits/2002/20050907colorado_dem_prty.pdf'),
+            # factories.AuditSearchViewFactory(audit_id=565),
+            # factories.AuditSearchViewFactory(far_release_date='2005-09-15'),
+        ]
+
+        filter_fields = (
+            ('finding_id', 3),
+            ('finding', 'Disclosure'),
+            ('issue_id', 220),
+            ('issue', 'Coordinated Expenditures'),
+            ('election_cycle', 2002),
+            ('committee_id', 'C00161786'),
+            ('committee_name', 'COLORADO DEMOCRATIC PARTY'),
+            ('committee_designation', 'U'),
+            ('committee_type', 'Y'),
+            ('committee_description', 'Party'),
+            ('candidate_id', ''),
+            ('candidate_name', ''),
+            # ('audit_case_id', 1203),
```

Comment:
I notice the these are still commented out, is this PR ready for review or is there still test work @pkfec and @rjayasekera ?

Reply:
@LindsayYoung 
Yes, i tested the existing filter fields, they worked as expected. Yes, this PR is ready for review.I am not sure, if we want to filter on other fields as well, which why i have commented those fields. If we decide not to have any filters, i can delete those lines.

Reply:
@pkfec I'm not sure I understand.  Issue (tiers), candidate name, committe ID, committee name and committee type should be filterable if we are trying to replicate the funcationality of the search system on classic.

Reply:
@PaulClark2 @Lindsay Yes, all those fields should be filterable.  And we can add them to API backend easily. We are awaiting to find out how the front end going to work, because records can be filter out and sort by front end logic also.  So it is matter of figuring out where the best place for the other filters.  Currently from API end point records can be filtered by finding, issue, election_cycle, cmte_id, cand_id.

Reply:
:wave: Just a note to double check you are pinging the right user. I've gotten 3 notifications on this thread, and I'm pretty sure that was unintentional.

Reply:
@lindsay we're very sorry about that, there was a bit of confusion on usernames!  Please feel free to remove yourself from any notifications on this; we'll try to do the same to clean things up.  Thanks!

Reply:
Sadly I'm afraid there's nothing we can do on our side, it's all on your end; as I said, please feel free to opt out of the review and notifications so you stop receiving things from us, apologies for the unexpected notifications!